backend/controllers/LocationsController.ts

```diff
@@ -0,0 +1,41 @@
+import { Response, Request } from "express";
+import Location from "../models/Locations";
+
+const LocationController = {
+  async getLocationId(req: Request, res: Response) {
+    try {
+      const { id } = req.body;
+
+      const locations = await Location.findById(id);
+
+      return res.json(locations);

```

Comment:
Maybe, we need to add a use-case when "id" from user will be incorrect ore not exist? And then throw 400 error.

Reply:
have fixed it